include/mosaic_gnss_driver/parsers/sbf/sbf.h

```diff
@@ -163,4 +163,30 @@ namespace sbf {
     };
 } // namespace sbf
 
+// CRC lookup table for 16-bit CRC for SBF blocks (makes the computation faster)
+// © Copyright 2020, Septentrio NV/SA.
+static const uint16_t CRC_LOOKUP_16[256] = {
```

Comment:
Should this be moved to src/core/parsers/sbf/sbf.cpp ? It is not required to be part of the public api.

Reply:
Yes. I don't think we will have to use the lookup table anywhere else, I guess.